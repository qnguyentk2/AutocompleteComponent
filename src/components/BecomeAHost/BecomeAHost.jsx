import React from 'react';
import BecomeAHostStyle from './BecomeAHost.scss';

const BecomeAHost = () => {
  return (
    <div className={BecomeAHostStyle.BecomeAHost}>
      <p>This it BecomeAHost page</p>
    </div>
  )
}

export default BecomeAHost;