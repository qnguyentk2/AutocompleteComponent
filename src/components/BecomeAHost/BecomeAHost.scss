.BecomeAHost {
  text-align: center;
  color: white;
  padding-top: 50px;
}