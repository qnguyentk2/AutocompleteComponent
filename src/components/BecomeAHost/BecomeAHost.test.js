/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
// import { MemoryRouter } from 'react-router';
import toJSON from 'enzyme-to-json';
import BecomeAHost from './BecomeAHost';

const propsData = {
}

const setup = props => {
  const wrapper = shallow(
    <BecomeAHost {...props} />
  );

  return { wrapper };
}

describe('Test PageLayout', () => {
  test('renders without crashing', () => {
    const { wrapper } = setup(propsData);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
})