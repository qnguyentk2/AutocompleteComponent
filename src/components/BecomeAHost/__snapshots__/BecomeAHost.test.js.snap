// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PageLayout renders without crashing 1`] = `
<div
  className="BecomeAHost"
>
  <p>
    This it BecomeAHost page
  </p>
</div>
`;
